src/index.tsx

```diff
@@ -794,6 +892,8 @@ const context = canvas.getContext("2d")!;
 // https://stackoverflow.com/questions/13879322/drawing-a-1px-thick-line-in-canvas-creates-a-2px-thick-line/13879402#comment90766599_13879402
 context.translate(0.5, 0.5);
 
+restore();
```

Comment:
<3

Reply:
This can be moved to the `componentDidMount` and then update component's state